corehq/apps/export/models/new.py

```diff
@@ -1610,6 +1610,24 @@ def get_value(self, domain, doc_id, doc, base_path, **kwargs):
                 values[state_index] = state.stock_on_hand
         return values
 
+
+class ConversionMeta(DocumentSchema):
+    path = StringProperty()
+    failure_reason = StringProperty()
+    info = ListProperty()
+
+
+class ExportMigrationMeta(Document):
```

Comment:
👍 good call. gonna do a couple dry runs on staging to see what other useful information would be good to add